plugins_overview.py

```diff
@@ -90,23 +89,25 @@ def add_all_items(method, dest, wrapopen=r'\(', wrapclose=r'\)'):
             'journals': [],
             'env': [],
     }
-    plugcontent = open(
-        os.path.join(PLUGDIR, plugfile)).read().replace('\n', '')
-    add_all_items(
-        "from sos.report.plugins import ",
-        plugs_data[plugname]['distros'],
-        wrapopen='',
-        wrapclose='(class|from|import)'
-    )
-    add_all_items("profiles = ", plugs_data[plugname]['profiles'], wrapopen='')
-    add_all_items("packages = ", plugs_data[plugname]['packages'], wrapopen='')
-    add_all_items("add_copy_spec", plugs_data[plugname]['copyspecs'])
-    add_all_items("add_forbidden_path", plugs_data[plugname]['forbidden'])
-    add_all_items("add_cmd_output", plugs_data[plugname]['commands'])
-    add_all_items("collect_cmd_output", plugs_data[plugname]['commands'])
-    add_all_items("add_service_status", plugs_data[plugname]['service_status'])
-    add_all_items("add_journal", plugs_data[plugname]['journals'])
-    add_all_items("add_env_var", plugs_data[plugname]['env'])
+    with open(os.path.join(PLUGDIR, plugfile)).read().replace('\n', '') as pfd:
+        add_all_items(
+            "from sos.report.plugins import ", plugs_data[plugname]['distros'],
+            pfd, wrapopen='', wrapclose='(class|from|import)'
```

Comment:
The "pfd" is an extra argument here, compared to current code - is that correct? (I need to understand that by myself as well)